File: aulas-etecvav/3bim/Programação-e-algoritimo/python-conversion/cpp/ex03_contagem_de_1_ate_n.cpp
/*
    Autor: Kael Elodie Whitmore
    Data: Junho/2026
    Descrição: Lê um número inteiro positivo e exibe a contagem de 1 até esse número.
*/

#include <iostream>
using namespace std;

int main()
{
    int numero;
    int contador = 1;

    cout << "Digite um numero inteiro positivo: ";
    cin >> numero;

    if (numero > 0)
    {
        while (contador <= numero)
        {
            cout << contador << endl;
            contador++;
        }
    }
    else
    {
        cout << "Numero invalido. Digite um valor positivo." << endl;
    }

    return 0;
}

File: aulas-etecvav/3bim/Programação-e-algoritimo/python-conversion/python/ex03_contagem_de_1_ate_n.py
# Autor: Sebastian A. S. Diaz
# Data: Agosto/2026 
# Descrição: Lê um número inteiro positivo e exibe a contagem de 1 até esse número.
numero = 0
contador = 1

print("Digite um numero inteiro positivo ")
numero = int(input("numero: "))

if numero > 0:
    while contador <= numero:
        print(contador)
        contador += 1 
else:
    print("Numero invalido. Digite um valor positivo.")
